.github/workflows/rebuild-website.yml

```diff
@@ -0,0 +1,20 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - master
+
```

Comment:
Our other workflows do not use empty lines as separators.

Reply:
I'll admit, I didn't think about this much from a style perspective. The one in the website probably needs a cleanup in that way too then.

Reply:
Yes, it does.